kmod/uoa/uoa.c

```diff
@@ -199,7 +199,11 @@ static int uoa_stats_percpu_show(struct seq_file *seq, void *arg)
         unsigned int start;
 
         do {
+#if (RHEL_MAJOR == 9 && RHEL_MINOR < 4)
```

Comment:
The `RHEL_MAJOR` macro may be undefined in not-redhat system. You should use linux kernel version macro `KERNEL_VERSION` instead.
`u64_stats_fetch_begin_irq()` and `u64_stats_fetch_retry_irq()` got obsoleted since linux v6.3